editor/editor_audio_buses.cpp

```diff
@@ -1189,6 +1202,13 @@ EditorAudioBuses::EditorAudioBuses() {
 	top_hb->add_child(_default);
 	_default->connect("pressed", this, "_load_default_layout");
 
+	_set_default = memnew(Button);
```

Comment:
I would not add a button for this. Changing from project settings should be enough.

Reply:
done!